hw/ip/prim/doc/prim_present.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Primitive Component: PRESENT Scrambler"
+---
+
+# Overview
+
+`prim_present` is an (unhardened) implementation of the encryption pass of the
```

Comment:
not a readme, so one sentence per line is recommended.

Reply:
actually, this might not be the right answer. are ip/prim docs rendered?
I think we were going to create a gathering master-doc for that ... My
memory is hazy on the plans here, remind me if you know.

Reply:
bhu... as far as I know, they are not linked anywhere yet (unless an actual doc refers to it, like alert handler -> lfsr for instance).

I'll convert to the compatible one-sentence-per-line format.